content/md/en/docs/tutorials/get-started/build-local-blockchain.md

```diff
@@ -165,22 +176,22 @@ To install the front-end template:
    If the command doesn’t return a version number, download and install `yarn` by running the following command:
```

Comment:
You can use `yarn version` and specify a version number to update `yarn` if you have an older version installed.